<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerTable</class>
 <widget class="QMainWindow" name="CustomerTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Franklin Gothic Book</family>
    <pointsize>12</pointsize>
    <italic>true</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Customer Table</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Pictures.qrc">
    <normaloff>:/picture/images/add-user.png</normaloff>:/picture/images/add-user.png</iconset>
  </property>
  <property name="toolTip">
   <string>Customer Table</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 0, 0);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="7">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>32</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Customer Table</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <widget class="QTableView" name="tableView_Customer">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>300</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0" colspan="7">
       <widget class="QLabel" name="label_id">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>32</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Update Customer Data</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="6" rowspan="5">
       <widget class="QLabel" name="label_picture">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_name">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_dev">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Phone</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_phone">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_genre">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_copies">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Address</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="5">
       <widget class="QLineEdit" name="lineEdit_address">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="8" column="0" rowspan="2">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>273</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="upload_customer_pic">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 0);</string>
        </property>
        <property name="text">
         <string>Upload Customer Picture</string>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="update_customer_pic">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 255, 0);</string>
        </property>
        <property name="text">
         <string>Update Customer Picture</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="update_customer_data">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(0, 255, 0);</string>
        </property>
        <property name="text">
         <string>Update Customer Data</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QPushButton" name="delete_customer_data">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(170, 0, 0);</string>
        </property>
        <property name="text">
         <string>Delete Customer Data</string>
        </property>
       </widget>
      </item>
      <item row="8" column="5">
       <widget class="QPushButton" name="home_page">
        <property name="font">
         <font>
          <family>Franklin Gothic Book</family>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Home page</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6" rowspan="2">
       <spacer name="verticalSpacer_7">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>273</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>241</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="2">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>241</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="3">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>241</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="4">
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>241</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="9" column="5">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>241</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="Pictures.qrc"/>
 </resources>
 <connections/>
</ui>
